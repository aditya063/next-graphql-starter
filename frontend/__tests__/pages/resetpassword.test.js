import React from 'react';
import { shallow } from 'enzyme';
import ResetPasswordPage from '../../pages/resetpassword';

describe('ResetPasswordPage', () => {
  let mockProps;
  let wrapper;

  beforeEach(() => {
    jest.resetAllMocks();
    mockProps = { children: [] };
    wrapper = shallow(<ResetPasswordPage {...mockProps} />, {
      disableLifecycleMethods: true
    });
  });

  it('matches snapshot', () => {
    expect(wrapper).toMatchSnapshot();
  });
});
