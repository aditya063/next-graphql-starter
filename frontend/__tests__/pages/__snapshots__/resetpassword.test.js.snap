// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ResetPasswordPage matches snapshot 1`] = `
<Component
  router={
    Object {
      "query": Object {
        "resetToken": "",
        "resetTokenExpiry": "",
      },
    }
  }
/>
`;
